docs/reference/config.md

```diff
@@ -1189,8 +1200,8 @@ The following settings are available:
   manifest.nextflowVersion = '!>=1.2'       // with ! prefix, stop execution if current version does not match required version.
   ```
 
-`manifest.organisation`
-: Project organisation
+`manifest.organization`
+: Project organization
```

Comment:
cc @christopher-hakkaart to check me on US / UK English in Nextflow docs

Reply:
Good spot. Docs should be in American English. The changes look good.

Reply:
-1 for this, it's it's annoying change existing setting only for spelling

Reply:
I see, ok